src/scopes/activityReport/index.js

```diff
@@ -13,6 +13,7 @@ import { withoutCalculatedStatus, withCalculatedStatus } from './calculatedStatu
 import { withProgramSpecialist, withoutProgramSpecialist } from './programSpecialist';
 import { withRole, withoutRole } from './role';
 import withRegion from './region';
+import { withoutProgramTypes, withProgramTypes } from './programType';
```

Comment:
Is there a reason that the import is "without" then "with", while use later is "with" then "without"?

Reply:
Just me letting VS Code's auto-import decide